### PR TITLE
Feature/visit

### PR DESCRIPTION
**Pull request by KostetskyiYaroslav**

### CODE REVIEW

**Comment by AndriiKatsubo** [comment]
Is file MedicalDocManagment.WebUI/Files/d0615814-8a96-43e0-bec6-83ed45d62aae.png necessary for this feature?

File: MedicalDocManagment.WebUI/MedicalDocManagment.WebUI.csproj [hunk 1]
```diff
@@ -103,9 +103,8 @@
       <HintPath>..\packages\Microsoft.Owin.Security.OAuth.2.1.0\lib\net45\Microsoft.Owin.Security.OAuth.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="MultipartDataMediaFormatter, Version=1.0.2.0, Culture=neutral, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>bin\MultipartDataMediaFormatter.dll</HintPath>
+    <Reference Include="MultipartDataMediaFormatter">
+      <HintPath>C:\Users\yakos\OneDrive\Документи\MultipartDataMediaFormatter.dll</HintPath>
```
Comment (AndriiKatsubo) [inline MedicalDocManagment.WebUI/MedicalDocManagment.WebUI.csproj:107]: Can it be changed to something less path dependent?

**Comment by AndriiKatsubo** [comment]
Where user can view added summary?

**Review by AndriiKatsubo — COMMENTED** [review]
Three questions have been made.

File: MedicalDocManagment.WebUI/MedicalDocManagment.WebUI.csproj (hunk 1 above)
Comment (KostetskyiYaroslav) [reply MedicalDocManagment.WebUI/MedicalDocManagment.WebUI.csproj:107]: I'll change it back

**Comment by KostetskyiYaroslav** [comment]
At the moment user can see it into the database only.
Task has goal to add possibility to create summary as a new visit to doctor.
> Where user can view added summary?

**Comment by KostetskyiYaroslav** [comment]
No, it doesn't. 
Thanks for comment.
 
> Is file MedicalDocManagment.WebUI/Files/d0615814-8a96-43e0-bec6-83ed45d62aae.png necessary for this feature?

**Pull request closed without merging**